temperature.js

```diff
@@ -0,0 +1,22 @@
+var Temperature = {
+     toFahrenheit (celsius) {
+         var fahrenheit;
+         fahrenheit = celsius * (9/5) + 32;
+         return fahrenheit;
+     },
+     toCelsius (fahrenheit) {
+         var celsius;
+         celsius = (fahrenheit - 32) * (5/9);
+         return celsius;
+     }
+//     toKelvin (celsius) {
+//         var kelvin;
+//         kelvin = (celsuis + 273.16) ;
+//         return kelvin;
+//     }
+ };
+ var tempToday = 82;
+ var tempInCelsius = Temperature.toCelsius(tempToday);
```

Comment:
Per the assignment instructions, `tempInCelsius` should be rounded to the nearest whole number. See [Math.round](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/round).